azimuth/dataset_split_manager.py

```diff
@@ -305,25 +311,26 @@ def add_tags(
 
     def get_tags(
         self, indices: Optional[List[int]] = None, table_key: Optional[PredictionTableKey] = None
-    ) -> List[Dict[str, bool]]:
+    ) -> Dict[int, Dict[Tag, bool]]:
         """Get tags from the dataset split.
 
         Args:
             indices: Set of indices to select.
             table_key: Predictions table to gather prediction tags.
 
         Returns:
-            List of records per index.
-
+             Value of tags per row_idx.
         """
         if indices is not None and len(indices) > 0:
             ds = self.get_dataset_split(table_key).select(indices)
         else:
             ds = self.get_dataset_split(table_key)
```

Comment:
I know it's not new code, but I couldn't find anything in your new code so... 😅
```suggestion
        ds = self.get_dataset_split(table_key)
        if indices is not None and len(indices) > 0:
            ds = ds.select(indices)
```